core/src/com/unciv/ui/screens/worldscreen/topbar/WorldScreenTopBar.kt

```diff
@@ -190,20 +192,31 @@ class WorldScreenTopBar(internal val worldScreen: WorldScreen) : Table() {
 
     private class SelectedCivilizationTable(worldScreen: WorldScreen) : Table(BaseScreen.skin) {
         private var selectedCiv = ""
+        // Instead of allowing tr() to insert the nation icon - we don't want it scaled with fontSizeMultiplier
+        private var selectedCivIcon = Group()
+        private val selectedCivIconCell: Cell<Group>
         private val selectedCivLabel = "".toLabel()
         private val menuButton = ImageGetter.getImage("OtherIcons/MenuIcon")
 
         init {
+            // vertically align the Nation name by ascender height without descender:
+            //  Normal vertical centering uses the entire font height, but that looks off here because there's
+            //  few descenders in the typical Nation name. So we calculate an estimate of the descender height
+            //  in world coordinates (25 is the Label font size set below), then, since the cells themselves
+            //  have no default padding, we remove that much padding from the top of this entire Table, and
+            //  give the Label that much top padding in return. Approximated since we're ignoring 'leading'.
+            val descenderHeight = Fonts.fontImplementation.getMetrics().run { descent / height } * 25f
```

Comment:
Please add `val fontSize = 25` above, and use `* fontSize` here and `.setFontSize(fontSize)` below